<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="1969" started-at="2018-09-30T07:39:28Z" finished-at="2018-09-30T07:39:30Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="1969" started-at="2018-09-30T07:39:28Z" finished-at="2018-09-30T07:39:30Z">
      <class name="com.googleRestApi.com.restAssured.GetStatusCodeTest">
        <test-method status="PASS" signature="setBaseUri()[pri:0, instance:com.googleRestApi.com.restAssured.GetStatusCodeTest@8bd1b6a]" name="setBaseUri" is-config="true" duration-ms="447" started-at="2018-09-30T13:09:28Z" finished-at="2018-09-30T13:09:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setBaseUri -->
        <test-method status="FAIL" signature="testStatusCode()[pri:0, instance:com.googleRestApi.com.restAssured.GetStatusCodeTest@8bd1b6a]" name="testStatusCode" duration-ms="1496" started-at="2018-09-30T13:09:29Z" finished-at="2018-09-30T13:09:30Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at com.googleRestApi.com.restAssured.GetStatusCodeTest.testStatusCode(GetStatusCodeTest.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testStatusCode -->
      </class> <!-- com.googleRestApi.com.restAssured.GetStatusCodeTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
